consent-store/consent_store.py

```
@@ -43,4 +46,4 @@ def health_check():
 
 if __name__ == "__main__":
     import uvicorn
-    uvicorn.run(app, host="0.0.0.0", port=8001)
\ No newline at end of file
+    uvicorn.run(app, host="0.0.0.0", port=CONSENT_STORE_PORT)
```

Ruff (0.11.9)

`CONSENT_STORE_PORT` may be undefined, or defined from star imports

(F405)